src/test/java/core/basesyntax/strategy/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.db.StorageImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.handlers.BalanceOperationHandler;
+import core.basesyntax.strategy.handlers.OperationHandler;
+import core.basesyntax.strategy.handlers.PurchaseOperationHandler;
+import core.basesyntax.strategy.handlers.ReturnOperationHandler;
+import core.basesyntax.strategy.handlers.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationStrategyImplTest {
+    private Storage storage;
+    private Map<FruitTransaction.Operation, OperationHandler> operationHandlerMap;
+
+    @Before
```

Comment:
You should use @BeforeClass instead of @Before
```suggestion
    @BeforeClass
```